### PR TITLE
[321] Show localized warning to the user when drive is not verified

### LINKED ISSUE #321: pop up information
When the sign has empty or wrong value, show pop up information: ‘Warning, this drive is not verified, contact with a person from PAH responsible for drives' (or similar)

### PR DESCRIPTION
**Pull request by arturtamborski**
Closes #321 :dancer: 

How to test it:
make start
submit drive without filling in signature field (the last one)
notice the warning is shown in form view and in drives list view

### CODE REVIEW

File: frontend/src/views/DrivesView.vue [hunk 1]
```diff
@@ -56,6 +56,13 @@
             <span class="font-weight-bold mr-1">{{ $t('drives.ending_mileage') }}</span>
             <span>{{ drive.endMileage }}</span>
           </p>
+          <div
+            v-if="!drive.isVerified"
+            class="alert alert-warning m-5"
```
Comment (arturtamborski) [inline frontend/src/views/DrivesView.vue]:
```suggestion
            class="alert alert-warning col-xs-12"
```

File: frontend/src/views/PassengerView.vue [hunk 2]
```diff
@@ -8,14 +8,20 @@
...
-        @change="syncToLocalStorage"
-        v-model.number="form.hash"
         id="hash"
         name="hash"
-        max="999999"
-        step="1"
+        type="text"
+        pattern="[0-9]+"
+        inputmode="numeric"
         min="0"
```
Comment (magul) [inline frontend/src/views/PassengerView.vue]: not supported in `type="text"`

File: frontend/src/views/PassengerView.vue [hunk 3]
```diff
@@ -8,14 +8,20 @@
...
       <input
-        type="number"
-        @change="syncToLocalStorage"
-        v-model.number="form.hash"
         id="hash"
         name="hash"
-        max="999999"
-        step="1"
+        type="text"
+        pattern="[0-9]+"
```
Comment (magul) [inline frontend/src/views/PassengerView.vue]: will `[0-9]{6}` work?

File: frontend/src/views/DriveFormView.vue [hunk 4]
```diff
@@ -200,10 +196,18 @@
...
-      @dismissed="confirmationffline = false"
+      @dismissed="confirmationOffline=false"
     >
       <b>{{ $t('drive_form.drive_added_offline_notification') }}</b>
     </b-alert>
+    <b-alert
+      class="col-xs-12"
+      variant="warning"
+      dismissible
+      :show="(confirmationOnline || confirmationOffline) && !isVerified"
```
Comment (magul) [inline frontend/src/views/DriveFormView.vue]: `confirmationOnline || confirmationOffline` -> `true`

File: frontend/src/views/DriveFormView.vue [hunk 5]
```diff
@@ -169,23 +169,19 @@
       <label for="signature">{{ $t('drive_form.signature') }}</label>
       <input
         id="signature"
-        type="number"
         name="signature"
-        min="0"
+        type="text"
+        pattern="[0-9]+"
```
Comment (pniedzwiedzinski) [inline frontend/src/views/DriveFormView.vue]:
```suggestion
        pattern="[0-9]{6}"
```

File: frontend/src/views/PassengerView.vue (hunk 3 above)
Comment (pniedzwiedzinski) [inline frontend/src/views/PassengerView.vue]:
```suggestion
        pattern="[0-9]{6}"
```

**Pull request merged**